docs/graph-homomorphism-as-natural-transformation.md

```diff
@@ -19,6 +23,12 @@ An example of this in haskell is Maybe.
  * The functor G(X) is Nothing.
  * The morphism between all F(X) and G(X) is Nothing (see [this issue though](https://github.com/matthewfranglen/category-set-graph/issues/1))
 
+[Paul: Let me translate this example into my language to see if I understand it properly.  The categories C and D are both *Set*, the functor F: *Set* -> *Set* is the identity (sends each object to itself and each morphism to itself), and the functor G: *Set* -> *Set* sends each set to the empty set and each function between sets to the empty function from the empty set to itself.  You propose to construct a natural transformation F -> G.
```

Comment:
Ahh, I see now - if `Nothing` is not the empty set but rather the singleton set whose only element is the empty set, then your construction is correct.  The singleton set is the terminal object in **Set** (and probably the subcategory of Haskell types), so it looks like this all works out.